### ISSUE TITLE
Error with vita file

### ISSUE THREAD

**Issue by SkylerPixel** [fjay69/UndertaleTools#6]
when i try to unpack the undertale vita file i get this error
![fail](https://user-images.githubusercontent.com/43079681/45245187-504a0900-b2c9-11e8-810c-a2f1d16f5e45.PNG)

**Comment by fjay69** [comment]
This is not valid game.win.

**Comment by SkylerPixel** [comment]
I just pulled it from my working Vita digital copy, weird I will redownload undertale and try again

**Comment by SkylerPixel** [comment]
i am just using vita shell with hacking to extract my game.win from my working undertale copy to my pc , am i doing something wrong

**Comment by fjay69** [comment]
Give me your game.win. I will check it. Also note that repacked game.win in some cases can cause crashes on Vita. I don't know how to fix this due I have not Vita.

**Comment by Mc-muffin** [comment]
Maybe you got the encrypted game.win? Fastest way to check is to open the file with a hexeditor and look at the 4 first bytes, they should be `46 4F 52 4D` or `FORM` in text mode.

**Comment by SkylerPixel** [comment]
so this is the game.win opened in a hex editor, i dont think it matches what Mc-muffin said so what does that mean, also here is a link to the game.win file https://drive.google.com/open?id=1nGuZlGrGbtNL2-FQG7pkre-xfgDSEmSQ

![capturehex](https://user-images.githubusercontent.com/43079681/45264963-9d85c200-b412-11e8-9461-598fadeaf0a5.PNG)

**Comment by Mc-muffin** [comment]
> so this is the game.win opened in a hex editor, i dont think it matches what Mc-muffin said so what does that mean, also here is a link to the game.win file https://drive.google.com/open?id=1nGuZlGrGbtNL2-FQG7pkre-xfgDSEmSQ
> 
> ![capturehex](https://user-images.githubusercontent.com/43079681/45264963-9d85c200-b412-11e8-9461-598fadeaf0a5.PNG)

Yep, you got the encrypted game.win (that means you are on a version higher than 3.60) you can use PSVimgtools to decrypt them or use Vitashell.

**Comment by SkylerPixel** [comment]
How do I use vitashell to decrypt the win file, I am a bit of a newb with Vita hacking stuff😅

**Comment by Mc-muffin** [comment]
You can follow [this guide](https://github.com/TheRadziu/NoNpDRM-modding/wiki#obtaining-decrypted-game-assets-from-nonpdrm-rips-through-vitashell-decryption-on-console) to decrypt the games directly, be noted that you need an FTP client beforehand (FileZilla would suffice) and in the last step you need to check the data.win (not the eboot.bin)
